### PR TITLE
Create 37-ipfs-conf-dev-meetings-and-user-registry.md

### PR DESCRIPTION
**Pull request by flyingzumwalt**
Please post ASAP.

### CODE REVIEW

File: content/post/37-ipfs-conf-dev-meetings-and-user-registry.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+date: 2018-05-19
+url: 37-ipfs-conf-dev-meetings-and-user-registry.md
+title: Announcements: IPFS Conf,  IPFS & libp2p Developer Meetings and a User Registry
```
Comment (VictorBjelkholm) [inline content/post/37-ipfs-conf-dev-meetings-and-user-registry.md]: This makes yml belive that you're trying to create a map, wrap the title in quotes and it'll work! 

```
title: "Announcements: IPFS Conf, IPFS & libp2p Developer Meetings and a User Registry" 
```

File: content/post/37-ipfs-conf-dev-meetings-and-user-registry.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+author: flyingzumwalt
+---
+
+We’re very excited to announce two events that have been in the making for a very long time. In (tentatively) November we will host a public **IPFS Conf** in Lisbon. To prepare for that event, in July we will convene small **IPFS and libp2p Developer Meetings** in Berlin. Both of these events will be loosely inspired by PyCon and Docker Conf — events where community members play a very active role in shaping the agenda, and where a large portion of time is allocated to working meetings where Working Groups and product teams make the most of being together in-person. 
+
+# IPFSConf in November
```
Comment (mgoelzer) [inline content/post/37-ipfs-conf-dev-meetings-and-user-registry.md]: Non-essential, but in every other section it's written "IPFS Conf" (with a space).

File: content/post/37-ipfs-conf-dev-meetings-and-user-registry.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+IPFSConf will be a large, open-registration event for the extended ecosystem of people and projects that touch IPFS, libp2p, IPLD and multiformats.  We are aiming to host that event in Lisbon in November 2018. The event is open to all. To pre-register for IPFSConf, use the [pre-registration form](https://goo.gl/forms/0Pu6VZzG8pRAmrrv2).
```
Comment (mgoelzer) [inline content/post/37-ipfs-conf-dev-meetings-and-user-registry.md]: "IPFS Conf" (add space)

Comment (mgoelzer) [inline content/post/37-ipfs-conf-dev-meetings-and-user-registry.md]: "IPFS Conf" (add space)

**Comment by mgoelzer** [comment]
LGTM

(I made two small suggestions regarding `s/IPFSConf/IPFS Conf/` but I feel it could be shipped as is.  Overall, nice work!)

**Comment by VictorBjelkholm** [comment]
@flyingzumwalt merge when you feel ready to publish it. As soon as it's in master, it'll be auto-deployed.

**Pull request merged**